web/src/popover_menus_data.js

```diff
@@ -129,6 +130,8 @@ export function get_topic_popover_content_context({stream_id, topic_name, url})
     const visibility_policy = user_topics.get_topic_visibility_policy(sub.stream_id, topic_name);
     const all_visibility_policies = user_topics.all_visibility_policies;
     const is_spectator = page_params.is_spectator;
+    const latest_msg_id = get_latest_message_id_in_topic(stream_id, topic_name);
+    url += `/with/${latest_msg_id}`;
```

Comment:
Hmm, I don't love having the code for constructing a URL split between two places; can we just have the caller construct this properly once?

Reply:
Updated!